server/liveblog/embed/embed_assets/default-theme/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "default-theme",
+    "version": "1.0",
+    "angularModule": "liveblog.default-theme",
+    "styles": [
+        "styles/embed.css",
+        "//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css"
+    ],
+    "scripts": [
+        "//code.angularjs.org/1.3.14/angular-sanitize.js",
+        "//code.angularjs.org/1.3.14/angular-animate.js",
+        "main.js"
```

Comment:
in this version you added main.js into the theme. in the main it was outside of the theme.
ex:
https://github.com/superdesk/liveblog/blob/master/server/liveblog/embed/assets/scripts/liveblog-embed/main.js
vs
https://github.com/superdesk/liveblog/blob/feature/theme-system/server/liveblog/embed/embed_assets/default-theme/main.js

so everything is ok. Still I don't like that the main controller sits in the theme. but this discussion is over, pakage.json LGTM.